bin/sdk/gutenberg.js

```diff
@@ -4,20 +4,52 @@
  */
 const fs = require( 'fs' );
 const path = require( 'path' );
+const forEach = require( 'lodash/forEach' );
 
 exports.config = ( { argv: { inputDir, outputDir }, getBaseConfig } ) => {
 	const baseConfig = getBaseConfig( {
 		cssFilename: '[name].css',
 		externalizeWordPressPackages: true,
 	} );
-	const editorScript = path.join( inputDir, 'editor.js' );
-	const viewScript = path.join( inputDir, 'view.js' );
+
+	const presetPath = path.join( inputDir, 'index.json' );
+	const utilsPath = path.join( inputDir, 'utils' );
+
+	let viewBlocksPoints;
+	let viewScriptEntry;
+	if ( fs.existsSync( presetPath ) ) {
+		const presetBlocks = require( presetPath );
+		// Find all the utils scripts
+		utilScripts = fs.readdirSync( utilsPath ).map( utilFile => path.join( utilsPath, utilFile ) ).filter( fs.existsSync );
```

Comment:
Note that in https://github.com/Automattic/wp-calypso/pull/28088 we introduced `gutenberg/extensions/presets/jetpack/utils/i18n` which is imported from each block separately, instead just once in the preset file.

Could we refactor utility-files by moving them to `/shared` and import them in each block that depends on them? That way it'll be easier to build each block separately with the SDK without relying on preset importing things such as jetpack category for us.

The purpose of the preset would thus be only to define combination of extensions; not new functionality per-se.

See also https://github.com/Automattic/wp-calypso/pull/27836 which adds `/shared` utility folder.

Reply:
Ohh funny, looks like I commented almost identical thing in resolved convo 😆  https://github.com/Automattic/wp-calypso/pull/28066/files#r228319972 — sorry I missed that.

Introduction of `utils/i18n` changes the situation again quite a bit, though.